### PR TITLE
Python 3 Porting (print statements and exceptions)

### PR DESCRIPTION
**Pull request by stevesdawg**
Updated print statements and exceptions to python 3 syntax.
Linked to issue: #256 

More PR's to follow, addressing more advanced python 3 compatibility issues.
Goal is to guarantee functionality on both python 2 and python 3 in this first round of PR's.

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -70,4 +70,4 @@ try:
 
     task_mod.run(options)
 except Exception as exception:
-    utils.admin(exception)
+    utils.admin(exception)
```
Comment (michaelblyons) [inline run:73]: keep the newline, maybe?

Comment (stevesdawg) [reply run]: Yep, added now

File: requirements.txt [hunk 2]
```diff
@@ -10,3 +10,4 @@ BeautifulSoup4
 mock
 xmltodict
 rtyaml
+future
```
Comment (JoshData) [inline requirements.txt:13]: I don't think this line is need for this PR, right?

Comment (stevesdawg) [reply requirements.txt:13]: @JoshData that's correct, snuck into this commit accidentally. I can remove to keep it clean and focused.

Comment (stevesdawg) [reply requirements.txt]: Should be fixed now

**Comment by stevesdawg** [comment]
Squashed the minor edits that @michaelblyons and @JoshData had into one commit to keep the history clean, just FYI.

**Comment by JoshData** [comment]
Thanks!

**Pull request merged**